hash/x68k_flop.xml

```diff
@@ -29649,6 +29684,20 @@ Expects to read track data from [FDC] drive 1, works if same disk loaded.
 		</part>
 	</software>
 
+	<!-- Title is confirmed in executable -->
+	<!-- Game plays in Tate mode -->
+	<software name="prote">
+		<description>Prote Pruste</description>
+		<year>1999</year>
+		<publisher>Soft Bank</publisher>
+		<info name="developer" value="ChinaSpark" />
```

Comment:
Is “Soft Bank” in this software list different to SoftBank (e.g. in fmtowns_cd.xml)?

Also, does this game have a title in Japanese script?